2_process.R

```diff
@@ -545,6 +557,9 @@ p2_targets_list <- list(
     p2_all_attr_SC_obs,
     left_join(p2_SC_observations, p2_all_attr, by = c('subsegid' = 'PRMS_segid', 'Date')) %>%
       rename(PRMS_segid = subsegid) %>%
-      filter(Date >= earliest_date)
+      filter(Date >= earliest_date) %>%
+      #remove bad sample (explained here https://github.com/USGS-R/drb-inland-salinity-ml/issues/190)
+      #This filter works to remove this point if mean_value is NA or > 25000 for this Date and PRMS_segid.
+      filter(!(Date == '1999-03-30' & PRMS_segid == '111_1' & mean_value > 25000))
```

Comment:
If we're already editing `p2_wqp_SC_filtered` in this PR, should we move this filtering step there? That way, we could be very explicit about which record we want to omit, rather than looking for a threshold value in `p2_all_attr_SC_obs`.

```r
... %>%
filter(!(MonitoringLocationIdentifier == "USGS-01421618" & ActivityStartDate == "1999-03-30"))
```

Either work and so I'm happy to leave this as is though.

Reply:
good idea to move to `p2_wqp_SC_filtered`